sdk/storage/storage-blob/src/utils/utils.common.ts

```diff
@@ -221,7 +221,10 @@ export function extractConnectionStringParts(connectionString: string): Connecti
     // SAS connection string
 
     const accountSas = getValueInConnString(connectionString, "SharedAccessSignature");
-    const accountName = getAccountNameFromUrl(blobEndpoint);
+    let accountName = getValueInConnString(connectionString, "AccountName");
+    if (!accountName) {
+      accountName = getAccountNameFromUrl(blobEndpoint);
+    }
```

Comment:
style nitpick

```suggestion
    const accountName = getValueInConnString(connectionString, "AccountName") ??getAccountNameFromUrl(blobEndpoint);
```

This is as long as you don't care about the case where `getValueInConnString` could return an empty string.

Reply:
So I double check, and actually, to not change behavior, we want to get account name from url if  `getValueInConnString` returns an empty string. I did not reallize that at first :)
So for now, I kept the `if()` and made this explicit with a comment. Let me know if that's ok to you